lib/FieldType/DataTransformer/SelectionValueTransformer.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\FieldType\DataTransformer;
+
+use eZ\Publish\Core\FieldType\Selection\Value;
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * DataTransformer for Selection\Value.
+ * Needed to display the form field correctly and transform it back to an appropriate value object,
+ * as single selection form fields return integer values, while multiselects return arrays.
+ */
+class SelectionValueTransformer implements DataTransformerInterface
```

Comment:
Agree, but for a different reason :) I don't see how the unit test as is could get any simpler - I think it might be too simple actually. Single select is not properly covered. And adding that would be a case for separate transformers.